### DISCUSSION TITLE
OC as a multirelay, comments and interest requested

### DISCUSSION THREAD

**Discussion by NikkiLacrima** [OpenCollarTeam/OpenCollar#1094] [Ideas]
Is there interest in and need for making OC a true multirelay that can handle several active RLV sources, like  RLV zones, cells and caves and active objects inside those. Please leave comments and ideas.

**Comment by o0Kirara0o** [comment]
I was going to make a request about this topic, but before to do it, i searched and found this conversation.
Yes, i think that a multirelay might be useful since a lot of people use a RLV zone with inside some RLV device and/or RLV cells that become useless if the RLV zone is using the unique channel the OC relay has. This mean that people should use another hud to have a multi relay.
So yes, i think it should be implemented something like the credita one maybe? (i don't remember the brand).
Also i don't think that 5 are needed, but mostly 3 relays in one are more than enough. One for the RLV zone, one for the cell and one in case of device inside the cell. I think 3 will cover most of the cases and also should keep the script lighter than a 5 one might do.

Let me add this too:
It's being a while since i was thinking such a suggestion, cause i had some of those cases and to be honest, use an external multirelay instead of the OC one, doesn't really thrill me
one reason because a conflict might happen, sl is still sl after all
another one is because i'm going to use another attachment point for an hud, and we all how short most of us are about attachment points :P
so, since most of us wear a collar 24/7, why we can't have such a feature in our own collar?

**Comment by SilkieSabra** [comment]
We consistently get requests and questions about the OC relay when it appears to fail in exactly this kind of situation.  Since Kirara has brought it up, let me say I believe we are working on doing this exact thing when it will become possible for V9 with LinkSetData.

**Comment by NikkiLacrima** [comment]
We have a multi relay ready to test, do you folks know of any good regions using RLV zones ??
or if you are mad enough to want to be a tester. let me know in world: Nikki.Lacrima

**Comment by NikkiLacrima** [comment]
Testable multirelay for oc 8.3 can be found here : Multirelay and LSD update for oc_relay.lsl #1160
